src/llvm.cr

```diff
@@ -90,7 +90,11 @@ module LLVM
     chars = LibLLVM.get_default_target_triple
     triple = string_and_dispose(chars)
     if triple =~ /x86_64-apple-macosx|x86_64-apple-darwin/
+      # normalize on `macosx` and remove minimum deployment target version
       "x86_64-apple-macosx"
+    elsif triple =~ /aarch64-apple-macosx|aarch64-apple-darwin/
+      # normalize on `macosx` and remove minimum deployment target version
+      "aarch64-apple-macosx"
     elsif triple =~ /aarch64-unknown-linux-android/
```

Comment:
`case` statement would be bit cleaner, I think.

Reply:
I'm actually planning a different refactor. Anyway, that's out of scope for this simple bug fix.